docs/source/markdown/podman-play-kube.1.md

```diff
@@ -64,6 +64,10 @@ Require HTTPS and verify certificates when contacting registries (default: true)
 then TLS verification will be used. If set to false, then TLS verification will not be used. If not specified,
 TLS verification will be used unless the target registry is listed as an insecure registry in registries.conf.
 
+#### **--create-only**, **-c**
```

Comment:
move this up to line 38 so the options will be listed alphabetically (with the exception of help)